### ISSUE TITLE
Undeclared dependency JSON::Validator::OpenAPI

### ISSUE THREAD

**Issue by eserte** [graphql-perl/GraphQL-Plugin-Convert-OpenAPI#4]
Test suite fails if JSON::Validator::OpenAPI is not installed:
```
#   Failed test 'use GraphQL::Plugin::Convert::OpenAPI;'
#   at t/00-compile.t line 4.
#     Tried to use 'GraphQL::Plugin::Convert::OpenAPI'.
#     Error:  Can't locate JSON/Validator/OpenAPI.pm in @INC (you may need to install the JSON::Validator::OpenAPI module) (@INC contains: ... .) at /home/cpansand/.cpan/build/2019020518/GraphQL-Plugin-Convert-OpenAPI-0.17-0/blib/lib/GraphQL/Plugin/Convert/OpenAPI.pm line 7.
# BEGIN failed--compilation aborted at /home/cpansand/.cpan/build/2019020518/GraphQL-Plugin-Convert-OpenAPI-0.17-0/blib/lib/GraphQL/Plugin/Convert/OpenAPI.pm line 7.
# Compilation failed in require at t/00-compile.t line 4.
# BEGIN failed--compilation aborted at t/00-compile.t line 4.
# Looks like you failed 1 test of 1.
t/00-compile.t ......... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests 
... (etc) ...
```

**Comment by mohawk2** [comment]
Thanks for this report! This is caused by https://github.com/mojolicious/json-validator/issues/145

**Comment by mohawk2** [comment]
Fixed, released as 0.18.
